Mathlib/CategoryTheory/Functor/KanExtension/Basic.lean

```diff
@@ -266,35 +268,145 @@ end
 
 section
 
-variable (L L' : C ⥤ D) (iso₁ : L ≅ L') (F F' : C ⥤ H) (e : D ≌ D') (iso₂ : F ≅ F')
+variable {L : C ⥤ D} {L' : C ⥤ D'} (G : D ⥤ D') (e : L ⋙ G ≅ L') (F : C ⥤ H)
 
-/-- The equivalence of categories `RightExtension (L ⋙ e.functor) F ≌ RightExtension L F`
-when `e` is an equivalence. -/
-noncomputable def rightExtensionEquivalenceOfPostcomp₁ :
-    RightExtension (L ⋙ e.functor) F ≌ RightExtension L F := by
-  have := CostructuredArrow.isEquivalence_pre ((whiskeringLeft D D' H).obj e.functor)
-    ((whiskeringLeft C D H).obj L) F
-  exact Functor.asEquivalence (CostructuredArrow.pre ((whiskeringLeft D D' H).obj e.functor)
-    ((whiskeringLeft C D H).obj L) F)
+/-- The functor `LeftExtension L' F ⥤ LeftExtension L F` induced by an isomorphism
+`L ⋙ G ≅ L'`. -/
+@[simps!]
+def LeftExtension.postcomp₁ : LeftExtension L' F ⥤ LeftExtension L F :=
+  StructuredArrow.map₂ (F := (whiskeringLeft D D' H).obj G) (G := 𝟭 _) (𝟙 _)
+    ((whiskeringLeft C D' H).map e.inv)
```

Comment:
Here we don't need an isomorphism of functors, and `e.inv` could be replaced by a natural transformation `L' ⟶ L ⋙ G`.

Reply:
Thanks for the suggestion, which I have implemented.